src/Invictus.Testing.Tests.Integration/LogicAppAuthenticationTests.cs

```diff
@@ -43,6 +44,32 @@ public async Task AuthenticateLogicAppManagement_UsingSecretProvider_Succeeds()
             Assert.True(_isClientSecretRequested);
         }
 
+        [Fact]
+        public async Task AuthenticateLogicAppManagement_UsingAccessToken_Succeeds()
```

Comment:
@tomkerkhove, added a test over here. Granted, it doesn't to that much but fetching a token and authenticating but I was thinking that an example where the token is fetched from within a pipeline and passed along is more suitable for a blogpost (#32)?

Reply:
I've missed that one, sorry! Blog post is ok but in theory you can shove it in our CI pipeline as well to ensure it still works.

If you want to do that in a seperate PR that's OK, then we'll just create an issue for that but feels like a must to verify the scenario imo, but might be just me.

Reply:
I agree, yet if we complete this and publish I can use the package to accomplish the setup on the project I'm currently working on. Then I can couple back from there with a working sample.
Suggestion: complete this, and let's create another issue assigned to me

Reply:
Why can't we just do it here? :) We don't need to ship a package if it's not verified.

Reply:
alright ;) I'll see if I get to it today but otherwise I'll take this up next week!

Reply:
updated this, please check